Lib/distutils/tests/test_spawn.py

```diff
@@ -87,11 +87,35 @@ def test_find_executable(self):
             rv = find_executable(dont_exist_program , path=tmp_dir)
             self.assertIsNone(rv)
 
-            # test os.defpath: missing PATH environment variable
+            # PATH=''
             with test_support.EnvironmentVarGuard() as env:
-                with mock.patch('distutils.spawn.os.defpath', tmp_dir):
-                    env.pop('PATH')
+                env['PATH'] = ''
```

Comment:
I think in order for this test to expose my comment above, it needs to have the `cwd` be inside `tmp_dir` (if you want to adjust this test to also catch that error) -- though maybe it would be a good idea to do that for all of the tests?

Reply:
I updated tests.